google/cloud/storage/tests/universe_domain_integration_test.cc

```diff
@@ -0,0 +1,98 @@
+// Copyright 2022 Google LLC
```

Comment:
nit: Copyright year to be 2024

Reply:
Thanks. This is what I get for copying and pasting 😆